src/entities/fractions/currencyAmount.ts

```diff
@@ -17,9 +16,10 @@ export class CurrencyAmount extends Fraction {
   /**
    * Helper that calls the constructor with the ETHER currency
    * @param amount ether amount in wei
+   * @param chainId
    */
-  public static ether(amount: BigintIsh): CurrencyAmount {
-    return new CurrencyAmount(CAVAX, amount)
+  public static ether(amount: BigintIsh, chainId: ChainId): CurrencyAmount {
```

Comment:
what if we init with default chainId like `chainId: ChainId = ChainId.AVALANCHE`, so it doesn't break existing code.